net/portmapper/portmapper_test.go

```diff
@@ -32,12 +32,13 @@ func TestClientProbe(t *testing.T) {
 		t.Skip("skipping test without HIT_NETWORK=1")
 	}
 	c := NewClient(t.Logf)
-	for i := 0; i < 2; i++ {
+	c.NewProber(context.Background())
+	for i := 0; i < 30; i++ {
 		if i > 0 {
 			time.Sleep(100 * time.Millisecond)
 		}
-		res, err := c.Probe(context.Background())
-		t.Logf("Got: %+v, %v", res, err)
+		res, err := c.Prober.CurrentStatus()
+		t.Logf("Got(t=%dms): %+v, %v", i*100, res, err)
```

Comment:
I notice that it discovers UPnP after 2.2 seconds on my machine, so I added the time in this test to make it clear when it finds it.